### PR TITLE
Add runbook for OS Places API key management

### PR DESCRIPTION
**Pull request by AntonyBishop**
Runbook that details process for how to create and revoke OS Places API keys

### CODE REVIEW

File: runbooks/source/os-places-api-key-management.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+# OS Places API Key Management
+
+The [OS Places API](https://developer.ordnancesurvey.co.uk/os-places-api) is the MoJ's product of choice for addess lookup and verification.
+
+Service Teams require a key to access the API.
+
+## Pre-requisites
+
+You must have access to the OS Places API portal. If not, please contact your [administrator](mailto:antony.bishop@digital.justice.gov.uk).
```
Comment (digitalronin) [inline runbooks/source/os-places-api-key-management.html.md.erb]: Is there any other email address we can use, which isn't tied to an individual person?

File: runbooks/source/os-places-api-key-management.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+> e.g. HMCTS Apply For Probate Online Production
+
+5. Check the "OS Names API" and "OS Places API - Capture and Verification" boxes.
+
+6. Click "Create Key".
+
+This will return you to the "My Keys" tab. Select the key you created from the list. You should copy the value for `Consumer Key`. This is the key that you will need to send to the requester.
+
+> Note - This key is a secret and should only be transfered via a secure method e.g. email of an encrypted archive with password in a seperate message.
```
Comment (digitalronin) [inline runbooks/source/os-places-api-key-management.html.md.erb]: "separate"

File: runbooks/source/os-places-api-key-management.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+4. Click "Save Key".
+
+This will disable access to the API. However, you will need to contact OS Places Support to permanently delete the key.
+
+5. Go to "Support" and click on "Contact Us".
+
+6. Complete the form ensuring you include details of the key you want to revoke.
+
+This process can take a couple of days for OS Places to revoke the key. You can check if it has been revoked by going back to the Portal. If the `Consumer Key` value is ~~struck through~~ then the key has been successfully revoked.
```
Comment (digitalronin) [inline runbooks/source/os-places-api-key-management.html.md.erb]: `~~struck through~~` should work, but it doesn't. You need to use `<del>struck through</del>` instead

File: runbooks/source/os-places-api-key-management.html.md.erb (hunk 1 above)
Comment (AntonyBishop) [reply runbooks/source/os-places-api-key-management.html.md.erb]: Not really. I could set up an "OS Places Admin" group? At the moment the only contact is me.

Comment (AntonyBishop) [reply runbooks/source/os-places-api-key-management.html.md.erb]: Group created. Have updated the email address.

**Pull request merged**